src/controller/FileBrowserController.vala

```diff
@@ -70,34 +70,34 @@ namespace BrickManager {
                             throw new IOError.PERMISSION_DENIED ("Cannot run files owned by system users.");
                         try {
                             var owner = file_info.get_attribute_string (FileAttribute.OWNER_USER);
+                            // openvt will unfortunately set the ownership of the new /dev/tty<num> to root:root
+                            // instead of root:tty. Even with root:tty the new application would fails to configure
+                            // the tty. We deal with that by changing the ownership of the new tty to belong to
+                            // the owner of the programm we launch.
+                            var ttyname_dir_template = "/tmp/brickman.XXXXXX".dup();
```

Comment:
Yes, also reported to the vala folks so that they can either fix that or at least document it.